runtime/entries/vue.js

```diff
@@ -17,7 +17,45 @@
  * under the License.
  */
 
-import setup from './setup'
-import * as Vue from 'weex-vue-framework'
+const instanceOptions = {}
 
-setup({ Vue })
+const Vue = {
+  createInstanceContext (instanceId, runtimeContext, data) {
+    const weex = runtimeContext.weex
+    const instance = instanceOptions[instanceId] = {
+      document: weex.document,
+      lifecycle: {}
+    }
+
+    const instanceContext = {
+      __weex_instance_id__: instanceId,
```

Comment:
The `__weex_instance_id__`, `__weex_document__`, `__weex_config__` can be get from the `weex` variable, there is no need to pass those apis to instance context.